dkc/core/tests/test_file_rest.py

```diff
@@ -16,3 +18,50 @@ def test_file_list_default_ordering(admin_api_client, folder, file_factory):
     resp = admin_api_client.get('/api/v2/files', data={'folder': folder.id})
     assert resp.status_code == 200
     assert [f['name'] for f in resp.data['results']] == ['A', 'B', 'C']
+
+
+@pytest.mark.django_db
+def test_file_rest_create_process(admin_api_client, folder):
+    """Test initialization of a file, without its blob."""
+    resp = admin_api_client.post(
+        '/api/v2/files',
+        data={
+            'folder': folder.id,
+            'name': 'test.txt',
+            'size': 42,
+        },
+    )
+    assert resp.status_code == 201
+    assert resp.data['size'] == 42
+    folder.refresh_from_db()
+    assert folder.size == 42
+    saved_file = File.objects.get(pk=resp.data['id'])
```

Comment:
```suggestion
    saved_file = File.objects.get(id=resp.data['id'])
```

In principle, the internal primary key field could be different than `id`. Since we're operating from the perspective of an external client here, which only knows about the concrete `id` field, that's what we should use to look up.

This isn't really important here, but just a thought for writing future code.